modules/ROOT/pages/custom-application-alerts.adoc

```diff
@@ -23,11 +22,11 @@ xref:deploying-to-cloudhub.adoc[Deploy to CloudHub]
 
 == How to Use Notifications
 
-Notifications give you the ability to give visibility into business related events inside your application. For example, you can raise notifications when:
+Notifications give you visibility into business related events inside your application. For example, you can create notifications for when:
 
 * Your application is unable to connect to a remote service
-* An error occurs which requires human intervention, such as a problem with data mapping
```

Comment:
lines 27, 28, and 29 are grammatically complete sentences so they should all have periods at the end?